packages/shared/src/models/templatedData/templatedData.ts

```diff
@@ -77,18 +73,18 @@ export class TemplatedData {
    * Main data conversion method
    * Iterate over data, parse string values and nested objects and arrays recursively
    */
-  public parse<T>(value: T = this.initialValue) {
+  public parse<T>(value: T) {
```

Comment:
additional bug fix - because the `TemplatedData` would use an initial value to parse could hit an infinite loop if the initial value contains an `undefined` child entry (trying to recursively parse would then re-populate the initial value each time). 

So have just made minor code modifications to not include the initialValue